include/onnxruntime/core/graph/graph.h

```diff
@@ -1033,6 +1033,18 @@ class Graph {
     }
   }
 
+  // Without removing the existing consumers, add a consumer to the give node arg name.
+  void AddConsumerNode(const std::string& node_arg_name, Node* consumer)
```

Comment:
Ideally, we shouldn't directly modify the node_arg_to_consumer_nodes_ list... 
User can easily forgot to call these two function after modifying the graph...

Reply:
The reason I am adding this is because, the alternative way to add a consumer node is a 3-step process
1. Get the existing nodes
2. Add/Remove the node to/from list
3. Update the consumer nodes
```
std::vector<Node*> nodes = graph.GetMutableConsumerNodes(name);
nodes.push_back(new_node); // or std::remove(nodes.begin(), nodes.end(), node);
graph.UpdateConsumerNodes(name, nodes);
```
I had it working but because I am making this change in a few places I though these new functions is more efficient.